src/plugins/tables/tables.js

```diff
@@ -134,7 +134,9 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 		paginate_buttons = $elm.next( ".bottom" ).find( ".paginate_button" ),
 		ol = document.createElement( "OL" ),
 		li = document.createElement( "LI" );
-
+		if ( !$elm.id ) {
```

Comment:
Move this block in the plugin initialization

Reply:
The block has been deleted. There is already a script that ensures, a unique id is assigned to the element